### PR TITLE
README

### PR DESCRIPTION
**Pull request by dncliss**
## Checklist before submitting

- [x ] Did you read the [contributor guide](https://github.com/open-ce/open-ce/blob/master/CONTRIBUTING.md)?
- [x ] Did you update any affected [documentation](https://github.com/open-ce/open-ce/blob/master/doc/)?
- [ ] Did you write any [tests](https://github.com/open-ce/open-ce/blob/master/tests/) to validate this change?  

## Description

This is a proposed draft of a **Generic** README file for the Open-CE feedstock repositories.  The central documentation will remain in the eponymous **open-ce** repository, with the feedstock README's providing brief context and then redirecting the reader to the main repository.

For consistency, we will use the same generic README for all feedstocks (except the name).

For that reason, I am creating this one as the one intended draft for approval.  Once the wording is agreed upon and approved, I'll push out the same README to the remaining feedstocks (with only the name changed).  This will avoid requiring dozens of individualized reviews, and will keep the most critical information central to the project repository.

### CODE REVIEW

**Comment by dncliss** [comment]
https://github.com/open-ce/open-ce/issues/48

File: README.md [hunk 1]
```diff
@@ -1 +1,9 @@
-# av-feedstock
\ No newline at end of file
+# av-feedstock
+  
+This repository is one of several that comprise the broader **Open-CE** project. Each feedstock repository provides a conda-based build framework for a specified package within **Open-CE**.
```
Comment (bethune-bryant) [inline README.md]:
```suggestion
This repository is one of several that comprise the broader [**Open-CE**](https://github.com/open-ce) project. Each feedstock repository provides a conda-based build framework for a specified package within [**Open-CE**](https://github.com/open-ce).
```

File: README.md [hunk 2]
```diff
@@ -1 +1,9 @@
...
+
+These feedstock repositories do not contain the source code for the corresponding packages themselves. Instead, they contain the automated means to build the desired package from the officially designated source location.
+
+The **Open-CE** project has a separate central controlling repository which coordinates the overall build, dependencies, and documentation requirements of the project. The central repository is named for the project itself, [open-ce](https://github.com/open-ce/open-ce).
```
Comment (bethune-bryant) [inline README.md]:
```suggestion
The [**Open-CE**](https://github.com/open-ce) project has a separate central controlling repository which coordinates the overall build, dependencies, and documentation requirements of the project. The central repository is named for the project itself, [open-ce](https://github.com/open-ce/open-ce).
```

**Review by bethune-bryant — COMMENTED** [review]
It might be useful to have `Open-CE` link to the organization page. I suggested changes for that above.

File: README.md (hunk 2 above)
Comment (dncliss) [reply README.md]: good idea; committing suggestion

**Review by jayfurmanek — APPROVED** [review]
This looks good! Thanks @dncliss !

**Review by bnemanich — APPROVED** [review]

**Review by bethune-bryant — APPROVED** [review]

**Pull request merged**